### PR TITLE
API support urls with more than one param after environment/endpoint

### PR DESCRIPTION
**Pull request by eladbash**
image

### CODE REVIEW

File: API/Api/Api.ts [hunk 1]
```diff
@@ -36,14 +43,13 @@ class WhiteToothAPI {
 
     parseRequestURL(url: string): ParsedRequest {
         const urlParts = url.split("/");
-        if (urlParts.length !== 3)
```
Comment (ChengaDev) [inline API/Api/Api.ts]: from one side - consider set this to some const with a meaningful name
from the other side - maybe this should be obvious..
so - your choise

File: API/Api/Api.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import express = require('express');
+import _ = require('lodash');
```
Comment (ChengaDev) [inline API/Api/Api.ts]: consider import only required methods?

File: API/Api/Api.ts [hunk 3]
```diff
@@ -19,12 +21,17 @@ class WhiteToothAPI {
         app.get('*', (req: express.Request, res: express.Response) => {
             try {
                 const parsedURL = this.parseRequestURL(req.url);
-                const environment = JSON.parse(this._environmentRepository.getEnvironmentByName(parsedURL.environment));
-                const response = environment.responses[parsedURL.url];
+                const environment = this._environmentRepository.getEnvironmentByName(parsedURL.environment);
+
+                if (_.isEmpty(environment) || _.isEmpty(environment.responses))
```
Comment (ChengaDev) [inline API/Api/Api.ts]: is this possible to have env without responses?
I remember we discussed about default responses

**Comment by ChengaDev** [comment]
Awesome man!
Long live WhiteTooth!

File: API/Api/Api.ts (hunk 2 above)
Comment (eladbash) [reply API/Api/Api.ts]: less important in server side I think

File: API/Api/Api.ts (hunk 3 above)
Comment (eladbash) [reply API/Api/Api.ts]: we can enforce it in the constructor if we get all environments and validate it - but I don't think it's efficient

**Pull request merged**